src/pages/ItemsList/components/FilterBar/components/FilterBarCheckList/FilterBarCheckList.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './filterBarCheckList.scss';
+
+function FilterBarCheckList({ category, listArr }) {
+  return (
+    <div className="category">
+      <div className="categoryHeader">
+        <div className="categoryText">{category}</div>
+        <img className="categoryIcon" src="./image/itemList/upArrow.png" />
+      </div>
+      <div className="CheckList">
```

Comment:
```suggestion
      <div className="checkList">
```
이름 첫글자에 대문자를 사용하는 건 컴포넌트에만 지정할 수 있는 컨벤션입니다

Reply:
완료